lib/color.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:flutter/material.dart';
+
+Color? primaryColor(int shade) {
```

Comment:
Why is this nullable?

Reply:
The `[]` returns a Color value from a map and if value doesn't exist, it returns null.